### PR TITLE
Generating Docs

### PR DESCRIPTION
**Pull request by Chizobaonorh**
Ready for review, successfully generated the docs.

### CODE REVIEW

File: seeds/breadboard/docs/.nojekyll [hunk 1]
```diff
@@ -0,0 +1 @@
+TypeDoc added this file to prevent GitHub Pages from using Jekyll. You can turn off this behavior by setting the `githubPages` option to false.
```
Comment (dglazkov) [inline seeds/breadboard/docs/.nojekyll:1]: Let's remove this file and set the `githubPages` option to `false`.

File: seeds/breadboard/docs/endpoint-api.md [hunk 2]
```diff
@@ -1,3 +0,0 @@
-# Breadboard Endpoint API Documentation
```
Comment (dglazkov) [inline seeds/breadboard/docs/endpoint-api.md:1]: ah -- we probably need to move this somewhere. I don't want to delete this file.

File: seeds/breadboard/docs/examples/node-example-2.js [hunk 3]
```diff
@@ -1,34 +0,0 @@
-/**
```
Comment (dglazkov) [inline seeds/breadboard/docs/examples/node-example-2.js:1]: Or that file.

Here's what I would suggest: let's change the TypeDoc out directory to point to `./docs/api`. This way, there won't be any conflicts.

File: seeds/breadboard/tsconfig.json [hunk 4]
```diff
@@ -4,5 +4,9 @@
     "outDir": "./dist"
   },
   "include": ["src/**/*", "tests/**/*"],
-  "extends": "@google-labs/tsconfig/base.json"
+  "extends": "@google-labs/tsconfig/base.json",
+  "typedocOptions": {
+    "entryPoints": "./src/index.ts",
+    "out": "docs",
```
Comment (dglazkov) [inline seeds/breadboard/tsconfig.json:10]: Change this to `docs/api` and re-run.

File: seeds/llm-starter/docs/.nojekyll [hunk 5]
```diff
@@ -0,0 +1 @@
+TypeDoc added this file to prevent GitHub Pages from using Jekyll. You can turn off this behavior by setting the `githubPages` option to false.
```
Comment (dglazkov) [inline seeds/llm-starter/docs/.nojekyll:1]: Remove that.

File: seeds/llm-starter/tsconfig.json [hunk 6]
```diff
@@ -4,5 +4,9 @@
     "outDir": "./dist"
   },
   "include": ["src/**/*", "tests/**/*"],
-  "extends": "@google-labs/tsconfig/base.json"
+  "extends": "@google-labs/tsconfig/base.json",
+  "typedocOptions": {
+    "entryPoints": "./src/index.ts",
+    "out": "docs",
```
Comment (dglazkov) [inline seeds/llm-starter/tsconfig.json:10]: Change this to `docs/api` and re-run.

**Review by dglazkov — COMMENTED** [review]
Almost there! To land, please change the typedocs settings to generate in `docs/api`, rather than `docs`. This way, it won't delete all the other doc files that are already in `docs`.

**Comment by Chizobaonorh** [comment]
okay let us do that

File: seeds/breadboard/docs/endpoint-api.md (hunk 2 above)
Comment (Chizobaonorh) [reply seeds/breadboard/docs/endpoint-api.md:1]: Where should i move it to?

Comment (dglazkov) [reply seeds/breadboard/docs/endpoint-api.md:1]: Oops, please disregard. This was a comment from earlier in the review process where I didn't realize that we needed to move the generated docs to `docs/api`.

**Pull request closed without merging**